README.md

```diff
@@ -70,6 +70,16 @@ Please, check the project [documentation](https://capsule.clastix.io) for the co
 
 Capsule is Open Source with Apache 2 license and any contribution is welcome.
 
+## Chart Development
+
+The documentation for each chart is done with [helm-docs](https://github.com/norwoodj/helm-docs). This way we can ensure that values are consistent with the chart documentation.
+
+We have a script on the repository which will execute the helm-docs docker container, so that you don't have to worry about downloading the binary etc. Simply execute the script (Bash compatible, might require sudo privileges):
```

Comment:
This is an important paragraph, thanks @oliverbaehler!
I think the root privileges shouldn't be necessary (and shall not). Why would they be?

Reply:
if you forgot to add ur working user to the docker group :smile:

Reply:
😆  I'd avoid mentioning sudo then, as I see it as a matter of having properly configured Docker for the working user.

Reply:
Totally agree, no reference on `sudo`.

Reply:
haha, i am one of those dudes

```
+ docker run -v /home/oliver/Projects/OSS/capsule:/helm-docs -u 1000 jnorwood/helm-docs:latest
docker: Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/create": dial unix /var/run/docker.sock: connect: permission denied.
```